Can't compile LLVM-instrumentation pass

Hello,
I'm trying to install AFLGo and followed the instructions on how to fuzz a binary with AFLGo. LLVM 4.0.0 is installed with cmake version 3.17.0 on Linux Mint 19.3 64-bit. I also tried it with LLVM 4.0.1 and got the same result. When trying to compile the code in aflgo/llvm_mode with "sudo make clean all" I get the following error message:

rm -f *.o *.so *~ a.out core core.[1-9][0-9]* test-instr .test-instr0 .test-instr1 
rm -f ../afl-clang-fast ../afl-llvm-pass.so ../afl-llvm-rt.o ../afl-llvm-rt-32.o ../afl-llvm-rt-64.o ../afl-clang-fast++
[*] Checking for working 'llvm-config'...
[*] Checking for working 'clang'...
[*] Checking for '../afl-showmap'...
[+] All set and ready to build.
clang -O3 -funroll-loops -Wall -D_FORTIFY_SOURCE=2 -g -Wno-pointer-sign -DAFL_PATH=\"/usr/local/lib/afl\" -DBIN_PATH=\"/usr/local/bin\" -DVERSION=\"2.52b\"  afl-clang-fast.c -o ../afl-clang-fast 
ln -sf afl-clang-fast ../afl-clang-fast++
clang++ `llvm-config --cxxflags` -Wl,-znodelete -fno-rtti -fpic -O3 -funroll-loops -Wall -D_FORTIFY_SOURCE=2 -g -Wno-pointer-sign -DVERSION=\"2.52b\" -Wno-variadic-macros -shared afl-llvm-pass.so.cc -o ../afl-llvm-pass.so `llvm-config --ldflags` 
warning: unknown warning option '-Wno-maybe-uninitialized'; did you mean '-Wno-uninitialized'? [-Wunknown-warning-option]
afl-llvm-pass.so.cc:397:24: error: no matching constructor for initialization of 'llvm::raw_fd_ostream'
        raw_fd_ostream cfgFile(cfgFileName, EC);
                       ^       ~~~~~~~~~~~~~~~
/usr/local/include/llvm/Support/raw_ostream.h:400:3: note: candidate constructor not viable: no known conversion from 'std::string'
      (aka 'basic_string<char>') to 'int' for 1st argument
  raw_fd_ostream(int fd, bool shouldClose, bool unbuffered=false);
  ^
/usr/local/include/llvm/Support/raw_ostream.h:357:7: note: candidate constructor (the implicit copy constructor) not viable:
      requires 1 argument, but 2 were provided
class raw_fd_ostream : public raw_pwrite_stream {
      ^
/usr/local/include/llvm/Support/raw_ostream.h:395:3: note: candidate constructor not viable: requires 3 arguments, but 2 were
      provided
  raw_fd_ostream(StringRef Filename, std::error_code &EC,
  ^
1 warning and 1 error generated.
Makefile:84: recipe for target '../afl-llvm-pass.so' failed

--- a/llvm_mode/afl-llvm-pass.so.cc
+++ b/llvm_mode/afl-llvm-pass.so.cc
@@ -394,7 +394,7 @@ bool AFLCoverage::runOnModule(Module &M) {
         /* Print CFG */
         std::string cfgFileName = dotfiles + "/cfg." + funcName + ".dot";
         std::error_code EC;
-        raw_fd_ostream cfgFile(cfgFileName, EC);
+        raw_fd_ostream cfgFile(cfgFileName, EC, sys::fs::F_None);
         if (!EC) {
           WriteGraph(cfgFile, &F, true);
         }

